src/ontology/cl-edit.owl

```diff
@@ -31277,6 +31279,29 @@ AnnotationAssertion(rdfs:label obo:CL_4033037 "mucus secreting cell of tracheobr
 EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
 SubClassOf(obo:CL_4033037 obo:CL_0002202)
 
+# Class: obo:CL_4033038 (lung resident memory CD4-positive, alpha-beta T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24064670") Annotation(oboInOwl:hasDbXref "PMID:31265968") obo:IAO_0000115 obo:CL_4033038 "An alpha-beta CD4 T cell that resides in the lung.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033038 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033038 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033038 "2023-04-27T10:15:45Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24064670") oboInOwl:hasExactSynonym obo:CL_4033038 "lung resident memory CD4 T cells")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31265968") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033038 "lung CD4-positive TRM")
+AnnotationAssertion(rdfs:label obo:CL_4033038 "lung resident memory CD4-positive, alpha-beta T cell")
+SubClassOf(obo:CL_4033038 obo:CL_0000897)
+SubClassOf(obo:CL_4033038 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002048))
+
+# Class: obo:CL_4033039 (lung resident memory CD8-positive, alpha-beta T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28418771") Annotation(oboInOwl:hasDbXref "PMID:31265968") Annotation(oboInOwl:hasDbXref "PMID:31558615") obo:IAO_0000115 obo:CL_4033039 "An alpha-beta CD8 T cell that resides in the lung. This cell expresses the resident marker CD69 and the chemokine receptor CXCR6, which regulates the localization in the lung.")
```

Comment:
CD69 and CXCR6 may not be specific to lung tissue, but rather any tissue-resident T cell so stating "which regulates the localization in the lung" may be misleading.